apps/aedevmode/src/aedevmode_app.erl

```diff
@@ -0,0 +1,28 @@
+-module(aedevmode_app).
+-behavior(application).
+
+-export([ start/2
+        , stop/1 ]).
+
+-export([post_check/0]).
+
+start(_StartType, _StartArgs) ->
+    aedevmode_sup:start_link().
+
+stop(_State) ->
+    lager:info("Stopping aedevmode_app", []),
+    ok.
+
+%% Run at 300, i.e. after normal env checks AND the db start
```

Comment:
I realize that it's not clear why I put this check so late in the flow.
Initially, I had it test for (dev mode AND `top_height() == 0`). Then I removed the top_height check ...